From 638627a19efa5a26ac1e6f7e64804010463d6d50 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Thu, 23 May 2024 15:48:07 -0700
Subject: [PATCH] ==== write-only monitoring ====

Implement a new monitoring primitive called vaddr-wo and paddr-wo.
Those are same to vaddr and paddr, but use soft-dirty feature for access
checking.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/qLG8W05V | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/qLG8W05V

diff --git a/damon_meta_changes/qLG8W05V b/damon_meta_changes/qLG8W05V
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.2

